endpoints/globus/endpoint_test.go

```diff
@@ -100,7 +100,7 @@ func TestGlobusStatus(t *testing.T) {
 	assert.NotNil(err)
 }
 
-// This runs setup, runs all tests, and does breakdown.
+// this runs setup, runs all tests, and does breakdown
```

Comment:
Vital capitalisation change!

Reply:
This is mainly to remind me and my OCD that comments are not formal prose, but it must look bizarre!